BLMM/BLMM.py

```diff
@@ -43,93 +46,18 @@ def _main(argv=None):
         voxelBatching = inputs['voxelBatching']
     else:
         voxelBatching = 0
-    
-    # --------------------------------------------------------------------------------
-    # Set up cluster
-    # --------------------------------------------------------------------------------
-    if 'clusterType' in inputs:
-
-        # Check if we are using a HTCondor cluster
-        if inputs['clusterType'].lower() == 'htcondor':
-
-            # Load the HTCondor Cluster
-            from dask_jobqueue import HTCondorCluster
-            cluster = HTCondorCluster()
-
-        # Check if we are using an LSF cluster
-        elif inputs['clusterType'].lower() == 'lsf':
-
-            # Load the LSF Cluster
-            from dask_jobqueue import LSFCluster
-            cluster = LSFCluster()
-
-        # Check if we are using a Moab cluster
-        elif inputs['clusterType'].lower() == 'moab':
-
-            # Load the Moab Cluster
-            from dask_jobqueue import MoabCluster
-            cluster = MoabCluster()
-
-        # Check if we are using a OAR cluster
-        elif inputs['clusterType'].lower() == 'oar':
-
-            # Load the OAR Cluster
-            from dask_jobqueue import OARCluster
-            cluster = OARCluster()
-
-        # Check if we are using a PBS cluster
-        elif inputs['clusterType'].lower() == 'pbs':
-
-            # Load the PBS Cluster
-            from dask_jobqueue import PBSCluster
-            cluster = PBSCluster()
-
-        # Check if we are using an SGE cluster
-        elif inputs['clusterType'].lower() == 'sge':
-
-            # Load the SGE Cluster
-            from dask_jobqueue import SGECluster
-            cluster = SGECluster()
-
-        # Check if we are using a SLURM cluster
-        elif inputs['clusterType'].lower() == 'slurm':
-
-            # Load the SLURM Cluster
-            from dask_jobqueue import SLURMCluster
-            cluster = SLURMCluster()
-
-        # Check if we are using a local cluster
-        elif inputs['clusterType'].lower() == 'local':
-
-            # Load the Local Cluster
-            from dask.distributed import LocalCluster
-            cluster = LocalCluster()
-
-        # Raise a value error if none of the above
-        else:
-            raise ValueError('The cluster type, ' + inputs['clusterType'] + ', is not recognized.')
-
-    else:
-        # Raise a value error if the cluster type was not specified
-        raise ValueError('Please specify "clusterType" in the inputs yaml.')
-    
-    # --------------------------------------------------------------------------------
-    # Connect to client
-    # --------------------------------------------------------------------------------
-    
-    # Connect to cluster
-    client = Client(cluster)   
+    maxmem = eval(inputs['MAXMEM'])
```

Comment:
Hi Vicente,

In the original BLMM code, if no MAXMEM memory limit is specified we assumed the default `MAXMEM` of `2**32` (following an old SPM convention). This used to happen in the `blmm_setup` code [here](https://github.com/TomMaullin/BLMM/blob/master/BLMM/src/blmm_setup.py#L124-L127). Once we had set this limit we saved it back to a "working version" of the inputs file [here](https://github.com/TomMaullin/BLMM/blob/master/BLMM/src/blmm_setup.py#L243-L245). However, in this PR we have read `MAXMEM` before setting the default. For this reason, I would suggest this should say:

```
if 'MAXMEM' in inputs:
        MAXMEM = eval(inputs['MAXMEM'])
else:
        MAXMEM = 2**32
```

Reply:
Sure, let me change it